BP/common/landed_titles/BP_empire_roman.txt

```diff
@@ -0,0 +1,12608 @@
+			add = @correct_culture_primary_score
+		}
+	}
+	
+	k_annonaria = {
+		color={ 220 20 60 }
+		capital = c_ravenna
+		
+		#Keep the ostrogoths et al. from changing away from k_italy
+		can_create = {
+			OR = {
+				ai = no
+				NAND = {
+					culture = ostrogothic
+					culture = skirian
+					culture = heruli
+					culture = romano_gothic
+					has_landed_title = k_ostrogoths
+				}
+				AND = {
+					highest_held_title_tier > tier_kingdom
+					OR = {
+						culture = ostrogothic
+						culture = skirian
+						culture = romano_gothic
+					}
+				}
+			}
+		}
+		
+		d_histria = {
+			color={ 54 167 156 }
+			color2={ 255 255 255 }
+			capital = c_aquileia
+			
+			c_pula = {
+				color = { 75 110 105 }
+				color2 = { 255 255 255 }
+				
+				cultural_names = {
+					roman = cn_pola
+					romano_gallic = cn_pola
+					romano_hispanic = cn_pola
+					romano_british = cn_pola
+					african_romance = cn_pola
+					raetoroman = cn_pola
+					thraco_roman = cn_pola
+					illyro_roman = cn_pola
+					pannonian = cn_pola
+					dalmatian = cn_pola
+					noricroman = cn_pola
+					romano_gothic = cn_pola
+					romano_dacian = cn_pola
+					romano_frank = cn_pola
+					romano_aramean = cn_pola
+					romano_vandalic = cn_pola
+					#Germanic
+					skirian = cn_pola
+					thuringian = cn_pola
+					ostrogothic = cn_pola
+				}
+				
+				b_pula = {
+					province = 2518
+					cultural_names = {
+						roman = cn_pola
+						romano_gallic = cn_pola
+						romano_hispanic = cn_pola
+						romano_british = cn_pola
+						african_romance = cn_pola
+						raetoroman = cn_pola
+						thraco_roman = cn_pola
+						illyro_roman = cn_pola
+						pannonian = cn_pola
+						dalmatian = cn_pola
+						noricroman = cn_pola
+						romano_gothic = cn_pola
+						romano_dacian = cn_pola
+						romano_frank = cn_pola
+						romano_aramean = cn_pola
+						romano_vandalic = cn_pola
+						#Germanic
+						skirian = cn_pola
+						thuringian = cn_pola
+						ostrogothic = cn_pola
+					}
+					color = { 75 110 105 }
+					color2 = { 255 255 255 }
+				}
+				b_piran = {
+					province = 2519
+					cultural_names = {
+						#Roman
+						roman = cn_piranum
+						romano_gallic = cn_piranum
+						romano_hispanic = cn_piranum
+						romano_british = cn_piranum
+						african_romance = cn_piranum
+						raetoroman = cn_piranum
+						thraco_roman = cn_piranum
+						illyro_roman = cn_piranum
+						pannonian = cn_piranum
+						dalmatian = cn_piranum
+						noricroman = cn_piranum
+						romano_gothic = cn_piranum
+						romano_dacian = cn_piranum
+						romano_frank = cn_piranum
+						romano_aramean = cn_piranum
+						romano_vandalic = cn_piranum
+						#Germanic
+						skirian = cn_piranum
+						thuringian = cn_piranum
+						ostrogothic = cn_piranum
+					}
+					color = { 75 110 105 }
+					color2 = { 255 255 255 }
+				}
+				b_pazin = {
+					province = 2520
+					cultural_names = {
+						#LT-Rasck: Piquentium matches Pazin Location better than Parentium
```

Comment:
Maybe, but Parentium is a far more important settlement. So unless you situate it outside the province it should be represented.

Reply:
I've made a note about this and will make the changes later today (hopefully).